### ISSUE TITLE
Some colors don't have enough contrast

### ISSUE THREAD

**Issue by crt0r** [huacat1017/huacat.pink-theme#2]
> First of all, thank you for developing such a good theme!

## I write a lot of documentation in Markdown for my work and sometimes have a hard time reading the text in a preview or an editor. So far, the issues are with blockquotes and some code blocks.

1. Some parts of the code block are nearly the same color as the background, so they are barely readable.
![photo_2022-10-21_11-23-33](https://user-images.githubusercontent.com/56602996/197151853-5e60ffb5-d29f-4e8d-ab5a-65c9e6c84714.jpg)
![photo_2022-10-21_11-24-54](https://user-images.githubusercontent.com/56602996/197151890-c25fb352-2dfb-4841-baf9-b90fb871d456.jpg)

1. The blockquote at the bottom of the screenshot is very light.
![photo_2022-10-21_11-21-34](https://user-images.githubusercontent.com/56602996/197151428-46f57244-5d70-4860-8ced-e0a95b709bbf.jpg)

**Comment by huacat1017** [comment]
First of all, Thank you for reporting these!
the code block issue seems like I don't color these.  But I don't know the elements now.
the second issue: It's the light green color in my color palette, maybe make it (blockquote texts) deeper or a new color to fix it.
I have no time working on this these days. I'll fix these on Nov 5 if there is no urgent homework.

**Comment by huacat1017** [comment]
Sorry for late replying. I’m not sure when I will be free to fix. My time management has been a mess. I’m sorry to delay and delay this. Hope to use my time better.

**Comment by crt0r** [comment]
> Sorry for late replying. I’m not sure when I will be free to fix. My time management has been a mess. I’m sorry to delay and delay this. Hope to use my time better.

Nothing serious. I have the same issues with my time due to college and day job. Especially because it's the end of the year and it's the period of final deadlines--for this year. Just focus on what's important for you first right now, Open Source comes next. 😸

**Comment by huacat1017** [comment]
I opened a new issue and mentioned this to VScode team. The code block problem is not as easy as I thought.
If you want to edit it yourself, go to "C:\Users\%USERNAME%\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\markdown-language-features\media\highlight.css" (Windows) to change the `.hljs-subst` class color by adding these lines on the file: (In my opinion, the color can fit few of light themes.) 
```
.vscode-light .hljs-subst,
.vscode-light .hljs-function,
.vscode-light .hljs-title,
.vscode-light .hljs-params,
.vscode-light .hljs-formula {
	color: #909090; /*Change the color to what you like*/
}
```
  
They use [highlight.js](https://highlightjs.org/) to get the Preview colored. You can check [this issue](https://github.com/microsoft/vscode/issues/91279). They recommended [Markdown Preview VS Code Highlighting](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-shiki) extension to make the syntax in the code blocks match active color theme because Shiki uses TextMate. (Shell script works well, but it only has two default themes for C++)  Syntax in Markdown Preview might be a problem for a long time. (OR you can go to Settings -> `markdown.styles` add the CSS yourself but it is unconvenient)
  
If you don't care about the background. You can try [Markdown Preview Enhanced](https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced) and [Markdown Preview Github Styling](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles) to avoid the text unable to read.

**Comment by crt0r** [comment]
> They use [highlight.js](https://highlightjs.org/) to get the Preview colored. You can check [this issue](https://github.com/microsoft/vscode/issues/91279). They recommended [Markdown Preview VS Code Highlighting](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-shiki) extension to make the syntax in the code blocks match active color theme because Shiki uses TextMate.

Thanks for the detailed reply! I'll try out these extensions.

**Comment by huacat1017** [comment]
I tried a darker color. Are you OK with this color?

| Before (#99ca82) | After (#3d8b1c)|
|---|---|
|<img width="182" alt="orginal green color" src="https://user-images.githubusercontent.com/57604590/214299752-1deef1b7-bdb5-403e-a059-95cc2d8a0c24.png"></img>|<img width="178" alt="modified sgreen color" src="https://user-images.githubusercontent.com/57604590/214299735-3d88fdb2-cb64-436f-9819-b7f0489381e4.png"></img>|

**Comment by crt0r** [comment]
> I tried a darker color. Are you OK with this color?
> Before (#99ca82) 	After (#3d8b1c)
> <img alt="orginal green color" width="182" src="https://user-images.githubusercontent.com/57604590/214299752-1deef1b7-bdb5-403e-a059-95cc2d8a0c24.png"> 	<img alt="modified sgreen color" width="178" src="https://user-images.githubusercontent.com/57604590/214299735-3d88fdb2-cb64-436f-9819-b7f0489381e4.png">

Yep, it's a lot more readable.

**Comment by huacat1017** [comment]
Published 0.4.1 to fix these. Enjoy!

**Comment by crt0r** [comment]
Thanks!
